src/dev/src/app/custom-props/custom-props.demo.module.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ClarityModule } from '@clr/angular';
+
+import { CustomPropsDemo } from './custom-props.demo';
+import { ROUTING } from './custom-props.routing';
+
+import '@clr/base/src/lib/components/clr-wc-element/clr-wc-element';
```

Comment:
I think this import path can be simplified based on the path map setup to

`import '@clr/base/clr-wc-element';`

If not something might not be quite right on the path mapping. Small nit we can fix this later as well once we have more components.